Qpf/Macro/Data/View.lean

```diff
@@ -122,11 +122,15 @@ def DataView.addDeclNameSuffix (view : DataView) (suffix : String) : DataView
 
 
 /-- Returns the fully applied form of the type to be defined -/
-def DataView.getExpectedType (view : DataView) : Term
-  := Syntax.mkApp (mkIdent view.shortDeclName) (
+def DataView.getExpectedTypeWithId (view : DataView) (id : Ident) : Term
+  := Syntax.mkApp id (
     (Macro.getBinderIdents view.binders.getArgs false)
   )  
 
+/-- Returns the fully applied form of the type to be defined -/
+def DataView.getExpectedType (view : DataView) : Term
+  := view.getExpectedTypeWithId (mkIdent view.shortDeclName)
+
 /-- Returns the fully applied, explicit (`@`) form of the type to be defined -/
 def DataView.getExplicitExpectedType (view : DataView) : CommandElabM Term
   :=  let args := Macro.getBinderIdents view.binders.getArgs true
```

Comment:
This function also has the wrong style, the `:=` should be on the same line as `CommandElabM Term`

(This is code you haven't touched in this PR, though, so feel free to ignore.

Also, I don't remember if I wrote this, but it is very possible. I wrote a bunch of this stuff before I knew what the style should've been, so it's not necessarily good to just mimic the style of what's already there. Sorry 'bout that)